packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.tsx

```diff
@@ -60,11 +55,13 @@ type Props = Partial<BpkCalendarGridProps> & {
   focusedDate?: Date | null;
   selectionConfiguration?: SelectionConfiguration;
   className?: string | null;
+  customRowHeight?: number;
```

Comment:
Might be worth adding a JSDOC comment to specify this value should be in `rem`. That's gonna reflect in the Storybook API docs then too automatically!

Reply:
Thanks - I opted for regular comments instead of JSDOC comments due to consistency (a lot of the other props are being specified by regular comments). But happy to change it if JSDOC is still the preference.

![Screenshot 2023-11-07 at 12 36 50](https://github.com/Skyscanner/backpack/assets/37026035/73077575-884c-464f-bfdd-7348f51e6ec6)

Reply:
no, that's totally fine!